ethcore/src/client/client.rs

```diff
@@ -2357,7 +2358,7 @@ impl ProvingBlockChainClient for Client {
 			self.engine.machine(),
 			&env_info,
 			self.factories.clone(),
-			false,
+			true,
```

Comment:
This will now use [transact_virtual](https://github.com/paritytech/parity-ethereum/blob/master/ethcore/src/executive.rs#L227l) to ensure that the account has enough balance for the execution

Reply:
What do you think about removing `virt` parameter from `state::prove_transaction` (always set to true) and add docs to `prove_transaction` trait method?